cspell.json

```diff
@@ -117,6 +117,7 @@
         "jsbi",
         "jsonml",
         "juicebox",
+        "kardiachain",
```

Comment:
```suggestion
```

Reply:
Why do you remove it? Could you explain me?

Reply:
Only one field is necessary.

Reply:
Ok bro, thanks for replying to me. I just removed it.